frontend/src/components/ConsultantRows.tsx

```diff
@@ -26,54 +28,54 @@ export default function ConsultantRows({
+            isListElementVisible
+              ? "border-l-secondary_default"
+              : isButtonHovered
+              ? "border-l-primary_default"
+              : "border-l-primary_l4"
+          } `}
+        >
+          <button
+            className={`p-2 rounded-lg ml-2 hover:bg-primary_default hover:bg-opacity-10 ${
+              isListElementVisible && "rotate-180"
+            }`}
+            onClick={toggleListElementVisibility}
+          >
+            <ChevronDown className={`text-primary_default w-6 h-6`} />
+          </button>
+        </td>
         <td>
-          <div className="flex flex-row gap-2 ">
-            <div
-              className={`py-0 w-0.5 rounded-lg ${
-                isButtonHovered ? "bg-primary_default" : "bg-primary_l4"
-              } ${isListElementVisible && "bg-secondary_default"}`}
-            ></div>
-            <div
-              className="flex flex-row gap-2 items-center h-[52px]"
-              onMouseEnter={() => setIsButtonHovered(true)}
-              onMouseLeave={() => setIsButtonHovered(false)}
+          <div className="flex flex-col gap-1 ">
+            <p
+              className={`text-black text-start ${
+                isListElementVisible ? "body-bold" : "body"
+              }`}
             >
-              <button
-                className={`p-2 rounded-lg hover:bg-primary_default hover:bg-opacity-10 ${
-                  isListElementVisible && "rotate-180"
-                }`}
-                onClick={toggleListElementVisibility}
-              >
-                <ChevronDown className={`text-primary_default w-6 h-6`} />
-              </button>
-              <div className="flex flex-col gap-1">
-                <p
-                  className={`text-black text-start ${
-                    isListElementVisible ? "body-bold" : "body"
-                  }`}
-                >
-                  {consultant.name}
-                </p>
-                <p className="detail text-neutral_l1 text-start">
-                  {`${consultant.yearsOfExperience} års erfaring`}
-                </p>
-              </div>
-            </div>
+              {consultant.name}
+            </p>
+            <p className="detail text-neutral_l1 text-start">
+              {`${consultant.yearsOfExperience} års erfaring`}
+            </p>
           </div>
         </td>
         {consultant.bookings?.map((b) => (
-          <td
-            key={b.weekNumber}
-            className={`px-2 py-1 rounded  ${
-              b.bookingModel.totalOverbooking > 0
-                ? `bg-black text-white`
-                : b.bookingModel.totalSellableTime > 0
-                ? `bg-semantic_4_l1`
-                : `bg-primary_l5`
-            }`}
-          >
-            <div className="flex flex-col gap-1">
+          <td key={b.weekNumber} className="h-[52px] p-0.5">
+            <div
+              className={`flex flex-col gap-1 p-2 justify-end rounded w-full h-full  ${
```

Comment:
Jeg vet h-full krever at parent har en fixed høyed for å fungere, så er litt usikker på om w-full har noe funksjon her